security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -709,8 +709,15 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse final
             doHandleRequest(finalResponse, proxiedResponse);
         } catch (IOException | ExecutionException | InterruptedException | TimeoutException e) {
             // connection problem with the host
-            logger.error("Exception occured when trying to connect to the remote host: ", e);
+            String errMsg = String.format("Exception occured when trying to connect to the remote url '%s'", sURL);
+            logger.error(errMsg, e);
             try {
+                if (e.getCause() instanceof java.net.UnknownHostException) {
```

Comment:
one could catch UnknownHostException first ?

this would implies doing something in the https://stackoverflow.com/questions/3526167/exception-handling-try-catch-inside-catch/3526225 taste, i.e. declaring HandleTrySomethingFailure.

Reply:
> one could also catch UnknownHostException first ?

I was thinking of something like that after having opened the PR, but we want to test the cause of the exception (e.getCause()), not the exception itself.